modules/cassandra/store/src/main/java/org/apache/ignite/cache/store/cassandra/persistence/PojoField.java

```diff
@@ -201,6 +201,8 @@ public Annotation getAnnotation(Class clazz) {
     public void setValueFromRow(Row row, Object obj, Serializer serializer) {
         if (calculatedField())
             return;
+        if (row.isNull(col))
```

Comment:
Spacing between semantic block is required by code style. So I suggest to insert new empty line before if.